spec/features/user_visits_problem_form_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'The problem form' do
+
+  it 'should visit problem form from when problem is selected' do
```

Comment:
really small thing but maybe remove the word 'from'?

Reply:
Done